securityscanutils/github_writer.go

```diff
@@ -74,7 +75,13 @@ func (g *GithubIssueWriter) getAllGithubIssues(ctx context.Context) ([]*github.I
 // Creates/Updates a Github Issue per image
 // The github issue will have the markdown table report of the image's vulnerabilities
 // example: https://github.com/solo-io/solo-projects/issues/2458
-func (g *GithubIssueWriter) CreateUpdateVulnerabilityIssue(ctx context.Context, release *github.RepositoryRelease, vulnerabilityMarkdown string) error {
+func (g *GithubIssueWriter) CreateUpdateVulnerabilityIssue(
+	ctx context.Context,
+	release *github.RepositoryRelease,
+	vulnerabilityMarkdown string,
+	developerDebugInstructions string,
+	outputLocalFilename string,
```

Comment:
I thought this was a good suggestion, so I refactored with a new `IssueWriter` interface with a few implementations. Since I created an `issuewriter` package and moved the `github_writer` implementation into it, I updated the changelog to specify `BREAKING_CHANGE`.